docs/release_notes.rst

```diff
@@ -20,6 +20,16 @@ Bug Fixes
 
 * Treat "package" as an imperative verb for D401 (#356).
 
+5.1.2 - September 13th, 2020
```

Comment:
Can you put this under the current development version please? I would like to update the version number and release date when I will actually release it :)

Reply:
I will try, it's my first time updating release notes.